app/src/main/res/layout/activity_member_profile.xml

```diff
@@ -10,16 +10,30 @@
         android:layout_width="match_parent"
         android:layout_height="wrap_content">
 
+        <de.hdodenhof.circleimageview.CircleImageView
+                android:id="@+id/circleImageView"
+                android:layout_width="96dp"
+                android:layout_height="96dp"
+                android:layout_marginTop="16dp"
+                android:layout_marginBottom="16dp"
```

Comment:
move the values to xml files following proper naming conventions

Reply:
extracted to `dimens.xml`